rio/subsystems/drivetrain.py

```diff
@@ -290,20 +302,20 @@ def getCurrentState(self):
         Returns the current state of this module.
         """
         # Current position of the motor encoder (in rotations)
-        encoder = self.steer_motor_encoder.getPosition()
+        self.encoder = self.steer_motor_encoder.getPosition()
 
         # if self.constants["steer_id"] == 1:
         # print(encoder)
 
         # Divide encoder by ratio of encoder rotations to wheel rotations, times 2pi
-        radians = encoder * (math.pi * 2)
+        self.radians = self.encoder * (math.pi * 2)
 
         # Construct a rotation2d object
-        rot = geometry.Rotation2d(radians)
+        self.rot = geometry.Rotation2d(self.radians)
```

Comment:
Why are these members? `self.`

Reply:
the self.s are needed otherwise the code doesn't work